lib/src/page_list_viewport_gestures.dart

```diff
@@ -198,10 +201,19 @@ class _PageListViewportGesturesState extends State<PageListViewportGestures> wit
     // Choose to lock in a particular axis direction, or not.
     final movementAngle = translation.direction;
     final movementAnglePositive = movementAngle.abs();
-    if (((2 / 6) * pi < movementAnglePositive) && (movementAnglePositive < (4 / 6) * pi)) {
+
+    // Consider axis locking for vertical and horizontal axes.
+    // The viewport thhresholds define the angle window around the
+    // vertical and horizontal axes which would result in axis locking.
+    // Vertical axis locking if the angle lays in the
+    // (pi/2 - hAngle, pi/2 + hAngle) window.
+    // Horizontal window is (vAngle, pi - vAngle).
```

Comment:
I'm confused by these ranges. This comment seems to suggest that vertical locking only happens when moving towards the left because it only references the half pointing towards `pi/2` radians and doesn't reference the half pointing towards `0.0` radians.

Then there's the comment about the horizontal locking which seems to be limited to a window based around the 45 degree line in the first quadrant. What about the other 3 quadrants of angles?

Reply:
Indeed confusing. I made a typo: vAngle and hAngle should be swapped.

Reply:
We are comparing absolute values of the movement angle. So for vertical we only check one bracket around +90 degrees and for the horizontal we check two brackets above the horizontal line.